apps/sandbox/android/app/build.gradle

```diff
@@ -0,0 +1,209 @@
+apply plugin: "com.android.application"
```

Comment:
I don't think we want to do any changes to sandbox app – let's add a sample module to bare-expo only

Reply:
If you're testing config plugins / prebuild with it let's use tests instead